docs/Using Fleet/MDM-migration-guide.md

```diff
@@ -7,45 +7,169 @@ This section provides instructions for migrating your hosts away from your old M
 
-You can do this in the Fleet UI by going to **Settings** > **Organization settings** >
-**Organization info** and adding a url to the desired image in the **Organization avatar URL (for
-dark backgrounds)** and **Organization avatar URL (for light backgrounds)** inputs. The appropriate
-image will show depending on the selected system theme.
+4. During the end user migration workflow, an end user's device will have their selected system theme (light or dark) applied. If your logo does not look good on either light or dark backgrounds, you can optionally set a logo for each theme:
+
+    1. Head to **Settings** > **Organization settings** >
+**Organization info** and add URLs to your logos in the **Organization avatar URL(for dark backgrounds)** and **Organization avatar URL (for light backgrounds)** fields.
+    
+    2. Select **Save**.
+
+fleetctl CLI:
+
+1. Create `fleet-config.yaml` file or add to your existing `config` YAML file:
+
+```yaml
+apiVersion: v1
+kind: config
+spec:
+  mdm:
+    macos_migration:
+      enable: true
+      mode: "voluntary"
+      webhook_url: "https://example.com"
+  ...
+```
+
+2. Fill in the above keys under the `mdm.macos_migration` key. 
+
+To learn about each option, in the Fleet UI, select the avatar on the right side of the top navigation, select **Settings > Integrations > Mobile device management (MDM)**, and scroll down to the **End user migration workflow** section.
+
+3. During the end user migration workflow, an end user's device will have their selected system theme (light or dark) applied. If your logo does not look good on either light or dark backgrounds, you can optionally set a logo for each theme:
+
+```yaml
+apiVersion: v1
+kind: config
+spec:
+  org_info:
+    org_logo_url: https://fleetdm.com/images/press-kit/fleet-blue-logo.png
+    org_logo_url_light_background: https://fleetdm.com/images/press-kit/fleet-white-logo.png
+  ...
+```
+
+Add URLs to your logo that looks good on dark background and light backgrounds in the `org_logo_url` and `org_logo_url_light_background` keys respectively.
+
+4. Run the fleetctl `apply -f fleet-config.yml` command to add your configuration.
+
+5. Confirm that your configuration was saved by running `fleetctl get config`.
+
+6. Send [these guided instructions](#how-to-turn-on-mdm-end-user) to your end users to complete the final few steps via Fleet Desktop.
+
+##### How to turn on MDM (end user)
+
+1. Select the Fleet icon in your menu bar and select **Migrate to Fleet**.
+
+2. Select **Start** in the **Migrate to Fleet** popup. 
+
+2. On your **My device** page, select **Turn on MDM** the button in the yellow banner and follow the instructions. 
+    * If you don’t see the yellow banner or the **Turn on MDM** button, select the purple **Refetch** button at the top of the page. 
+    * If you still don't see the **Turn on MDM** button or the **My device** page presents you with an error, please contact your IT administrator.
+
+## Migration report
```

Comment:
This section is great. It answers the "did it work?" question. I wonder if we can be more explicit here. "Migration report" is so bland and abstract. "How do I check the progress of my migration?" "Checking migration progress" etc etc. Let's use clear everyday language as much as possible.

Reply:
Agree it's bland. Updated the docs to use "Check migration progress: